client/models/section.coffee

```diff
@@ -45,4 +45,12 @@ module.exports = class Section extends Backbone.Model
     if @get('type') is 'artworks'
       _.extend super, ids: @artworks.pluck '_id'
     else
-      super
\ No newline at end of file
+      super
+
+   getLayout: ->
+    layout = 'column_width'
+    if @get('type') is 'text'
+      layout = 'overflow_fillwidth' if @get('body')?.includes('<blockquote>')
+    else if @get('layout')
+      layout = @get('layout')
+    return layout
```

Comment:
Oh I see- was mostly looking at the standard when building and didn't notice the detail- yeah probably easiest to make one more layout size